.github/workflows/codeql-analysis.yml

```diff
@@ -14,9 +14,13 @@ name: "CodeQL"
 on:
   push:
     branches: [ main ]
+    paths-ignore:
+      - "./website/**/*.md"
```

Comment:
I'm totally open to removing this change from the PR. I just noticed that the PR had some long running actions because it was executing the compilation and CodeQL steps on code changes that don't affect the actual C# and Hot Chocolate source code. I figured removing the markdown files from the CodeQL workflow's scope would help speed up some builds and PRs, but if it's preferred to keep it enabled I can revert this change.